sdk/src/loom/Platform/GameController.ls

```diff
@@ -0,0 +1,160 @@
+        public static var AXIS_LEFTY:uint        = 1;
+        public static var AXIS_RIGHTX:uint       = 2;
+        public static var AXIS_RIGHTY:uint       = 3;
+        public static var AXIS_TRIGGERLEFT:uint  = 4;
+        public static var AXIS_TRIGGERRIGHT:uint = 5;
+        /**
+         * AXIS_COUNT is not a valid axis by itself
+         * It is used to help iterating through axes
+         */
+        public static var AXIS_MAX:uint = 6;
+        
+        /** Specifies the number of connected game controllers. */
+        public static native var numControllers:int;
+        
+        /** Name of a connected controller. */
+        public native var name:String;
+        
+        /** Delegate for handling button events. */
+        public native var onButtonEvent:ButtonDelegate;
+        /** Delegate for handling axis events. */
+        public native var onAxisEvent:AxisDelegate;
+        
+        /**
+         * Returns a GameController object.
+         * 
+         * @param index The index of the controller
+         * @return Returns a GameController if it is connected
+         */
+        public static native function getGameController(index:int = -1):GameController;
+        
+        /**
+         * Checks if game controller is haptic.
+         * 
+         * @param device The id of game controller.
+         * @return True if game controller is haptic, false if it is not haptic.
+         */
+        public native function isHaptic():Boolean;
+        
+        /**
+         * Causes a game controller to start vibrating.
+         * 
+         * @param device The id of game controller.
+         * @param intensity Intensity of vibration ranging from 0 to 1.
+         * @param ms Duration of vibration in miliseconds. Accepts GameController.RUMBLE_INFINITY for infinite duration.
+         */
+        public native function startRumble(intensity:Number, ms:uint);
+        
+        /**
+         * Causes a game controller to stop vibrating.
+         * 
+         * @param device The id of game controller.
+         */
+        public native function stopRumble();
+        
+        /**
+         * Gets the state of a specific button.
+         * 
+         * @param button The id of wanted button
+         * @return Returns true if button is pressed, otherwise returns false.
+         */
+        public native function getButton(button:uint):Boolean;
+        
+        /**
+         * Gets the value of queried axis.
+         * 
+         * @param axis The id of wanted axis
+         * @return Returns a number between -32768 and 32767
+         */
+        public native function getAxis(axis:uint):int;
+        
+        /**
+         * Gets the value of queried axis. The value is converted to a number ranging from -1 to 1 for usability.
+         * 
+         * @param axis The id of wanted axis
+         * @return Returns a range between -1 and 1
+         */
+        public function getAxis2(axis:uint):Number
```

Comment:
Renamed to `getNormalizedAxis`. Although I can't really decide which would be better: `getNormalizedAxis` or `getAxisNormalized`